modules/database/src/admin/schema.admin.ts

```diff
@@ -38,45 +38,57 @@ export class SchemaAdmin {
             updatedAt: 1,
           },
         },
-      )
-      .then(r => {
-        return { schemas: r };
-      });
+      );
+    const cndSchemas = await this.database
+      .getSchemaModel('_DeclaredSchema')
+      .model.findMany(
+        { 'modelOptions.conduit.cms': { $exists: false } },
+        {
+          select: 'name extensions',
+          sort: {
+            updatedAt: 1,
+          },
+        },
+      );
+    return { schemas: cmsSchemas.concat(cndSchemas) };
   }
 
-  async importCustomSchemas(call: ParsedRouterRequest): Promise<UnparsedRouterResponse> {
+  async importSchemas(call: ParsedRouterRequest): Promise<UnparsedRouterResponse> {
     const schemas = call.request.params.schemas;
     for (const schema of schemas) {
-      const existingSchema = await this.database
-        .getSchemaModel('_DeclaredSchema')
-        .model.findOne({ name: schema.name });
-      const operation = isNil(existingSchema) ? 'create' : 'update';
-      const imported = schema.modelOptions.conduit.imported === true;
-      const modelOptions = SchemaConverter.getModelOptions({
-        cmsSchema: true,
-        existingModelOptions: schema.modelOptions,
-        importedSchema: imported,
-      });
-      try {
-        validateSchemaInput(schema.name, schema.fields, modelOptions);
-      } catch (err: unknown) {
-        throw new GrpcError(status.INTERNAL, (err as Error).message);
+      const isCmsSchema = schema.modelOptions;
+      if (isCmsSchema) {
+        const existingSchema = await this.database
+          .getSchemaModel('_DeclaredSchema')
+          .model.findOne({ name: schema.name });
+        const operation = isNil(existingSchema) ? 'create' : 'update';
+        const imported = schema.modelOptions.conduit.imported;
+        const modelOptions = SchemaConverter.getModelOptions({
+          cmsSchema: true,
+          existingModelOptions: schema.modelOptions,
+          importedSchema: imported,
+        });
+        try {
+          validateSchemaInput(schema.name, schema.fields, modelOptions);
+        } catch (err: unknown) {
+          throw new GrpcError(status.INTERNAL, (err as Error).message);
+        }
+        await this.schemaController.createSchema(
+          new ConduitSchema(
+            schema.name,
+            schema.fields,
+            modelOptions,
+            schema.collectionName,
+          ),
+          operation,
+          imported,
+        );
       }
-      await this.schemaController.createSchema(
-        new ConduitSchema(
-          schema.name,
-          schema.fields,
-          modelOptions,
-          schema.collectionName,
-        ),
-        operation,
-        imported,
-      );
       if (schema.extensions.length > 0) {
         for (const extension of schema.extensions) {
           await this.database.setSchemaExtension(
             schema.name,
-            extension.owner,
+            extension.ownerModule,
```

Comment:
This is technically breaking the `API` (`owner` -> `ownerModule`).
But it seems like the export endpoint actually returns `ownerModule`.
As such, I'm fine with this, but let's document in PR (include breaking change in conventional commit)

Reply:
This isn't a breaking change, because as you said, the export returns the extensions as they are stored in a schemas (``ownerModule``), meaning the result of export doesn't change at all. It would be a breaking change if the extensions used to be stored in schemas with ``owner`` (but as I checked, they were always ``ownerModule`` so this is just a bug fix)

Reply:
It's a breaking change regardless of whether it's a fix or not.
API behaved in one way, now it doesn't anymore.

But this isn't really a front-facing endpoint, so we shouldn't worry too much.